share/dkg/pedersen/dkg.go

```diff
@@ -1131,28 +1130,32 @@ func GetNonce() []byte {
 }
 
 func (d *DistKeyGenerator) sign(p Packet) ([]byte, error) {
-	msg := p.Hash()
+	msg, err := p.Hash()
+	if err != nil {
+		return nil, err
+	}
+
 	priv := d.c.Longterm
 	return d.c.Auth.Sign(priv, msg)
 }
 
 func (d *DistKeyGenerator) Info(keyvals ...interface{}) {
-	d.c.Info(append([]interface{}{"generator"}, keyvals...))
+	d.c.Info("generator", keyvals)
 }
 
 func (d *DistKeyGenerator) Error(keyvals ...interface{}) {
-	d.c.Info(append([]interface{}{"generator"}, keyvals...))
+	d.c.Info("generator", keyvals)
 }
 
 func (c *Config) Info(keyvals ...interface{}) {
 	if c.Log != nil {
-		c.Log.Info(append([]interface{}{"dkg-log"}, keyvals...))
+		c.Log.Info("dkg-log", keyvals)
 	}
 }
 
 func (c *Config) Error(keyvals ...interface{}) {
 	if c.Log != nil {
-		c.Log.Error(append([]interface{}{"dkg-log"}, keyvals...))
+		c.Log.Error("dkg-log", keyvals)
```

Comment:
I think I remember adding these at some point because our logger weren't working properly otherwise.

I'll need to test this locally actually with a couple loggers I guess T.T